File: 3533-snake-in-matrix/snake-in-matrix.cpp
class Solution {
public:
    int finalPositionOfSnake(int n, vector<string>& commands) {
        int row=0;
        int column=0;
        for (string path:commands){
            if (path=="RIGHT"){
                column+=1;
            }else if (path=="LEFT"){
                column-=1;
            }else if (path=="UP"){
                row-=1;
            }else{
                row+=1;
            }
        }
        return (row*n)+column;
    }
};

File: 3533-snake-in-matrix/snake-in-matrix.py
class Solution(object):
    def finalPositionOfSnake(self, n, commands):
        """
        :type n: int
        :type commands: List[str]
        :rtype: int
        """
        row=column=0
        for path in commands:
            if path=="RIGHT":
                column+=1
            elif path=="LEFT":
                column-=1
            elif path=="UP":
                row-=1
            else:
                row+=1

        return (row*n)+column                    